.storybook/preview-head.html

```diff
@@ -1,3 +1,9 @@
 <style>
   @import url('https://rsms.me/inter/inter.css');
```

Comment:
Seems we have hard dep on external resource that just a blog of some guy. Maybe change it to Google Fonts? https://fonts.google.com/specimen/Inter

Reply:
Do we ever need this? @tenphi ?

Reply:
When I added this line there was no Inter on the google fonts. So yes, we can change the import now. Make sure It will not import too much :)

Reply:
@tenphi @JiLiZART added task for that https://cubedevinc.atlassian.net/browse/CUK-76